### ISSUE TITLE
Reference to SSM Parameter Store values in the Parameters.yml file

### ISSUE THREAD

**Issue by onema** [LambdaSharp/LambdaSharpTool#96]
## Background
The parameter files are used to fill in Module.yml parameters at deployment time. Currently, there is no way to reference configuration values that are located in external systems such as SSM Parameter Store

## Possible solution
Enable LambdaSharp to resolve references in the parameters file to SSM parameter store at deployment time.

### Example
```yaml
# Module.yml
Items:
 - Parameter: Foo
 - Parameter: Bar
   Type: Secret
 - Parameter: Baz
   Default: "something" 
```

```yaml
# Parameters.yml

Foo: !GetParam /path/to/Foo
Bar: !GetParam /path/to/Bar
Baz: "Something Else"
```

The parameter references should be tier agnostic, and the tool should be able to resolve the proper parameter-store name at deployment time.

**Comment by bjorg** [comment]
I also added support for `!GetEnv` and `!GetConfig`. The latter can read entries from a JSON configuration file.

**Comment by bjorg** [comment]
Released in v0.5.0.3
